BUILD.md

```diff
@@ -0,0 +1,65 @@
+# Building the project
+
+We use [Gradle build flavors](http://developer.android.com/tools/building/configuring-gradle.html#workBuildVariants) to enable a number of different build variants of MUSER.
+
+For now, We have one  Gradle "platform" flavor dimensions:
+
+* **Google** = Normal Google Play release
+
+...and one Gradle "brand" flavor dimensions:
+
+* **free** = Original MUSER brand
+* **debug** = Debug variant of the MUSER
+
+We have one build variant with one release and one debug type resulting in 2 build variants. 
+
+The below instructions assume you're going to be building for the **Google** platform flavor and original **debug** brand by default (e.g., `devDebug`).
+
+### Prerequisites for both Android Studio and Gradle
+
+1. Clone this repository
+1. Install [Java Development Kit (JDK)](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
+
+### Building in Android Studio
+
+1. Download, install, and run the latest version of [Android Studio](http://developer.android.com/sdk/installing/studio.html).
+1. At the welcome screen select `Import Project`, browse to the location of this repository and select it then select Ok.
+1. Open the Android SDK Manager (Tools->Android->SDK Manager) and add a checkmark for the necessary API level (see `compileSdkVersion` in [`MUSER/build.gradle`](MUSER/build.gradle)) then select OK.
+1. Connect a [debugging enabled](https://developer.android.com/tools/device.html) Android device to your computer or setup an Android Virtual Device (Tools->Andorid->AVD Manager).
+1. Open the "Build Variants" window (it appears as a vertical button on left side of workspace by default) & choose **devDebug** to select the Google Play version.
+1. Click the green play button (or Alt+Shift+F10) to build and run the project!
+
+### Building from the command line using Gradle
+
+1. Set the `JAVA_HOME` environmental variables to point to your JDK folder (e.g. `C:\Program Files\Java\jdk1.6.0_27`)
```

Comment:
We want people using JDK 7 or higher, so I'd prefer to reference 7 instead of 6 in the example:

```suggestion
1. Set the `JAVA_HOME` environmental variables to point to your JDK folder (e.g. `C:\Program Files\Java\jdk1.7.0_27`)
```